src/app/main.cpp

```diff
@@ -30,6 +30,13 @@ int main(int argc, char ** argv) {
 
     // Create window.
     QApplication app(argc, argv);
+
```

Comment:
- [ ] Please move this change into new separate commit.
- [ ] Please provide conventional commit message to this commit: https://www.conventionalcommits.org/en/v1.0.0/ e.g. "feat: add organization name and domain to construct settings path"